src/app/Middleware/RequestLogger/loggerOptions.cs

```diff
@@ -9,6 +9,6 @@ public class LoggerOptions
         public bool Log3xx { get; set; } = true;
         public bool Log4xx { get; set; } = true;
         public bool Log5xx { get; set; } = true;
-        public double TargetMs { get; set; } = 250;
+        public double TargetMs { get; set; } = 1000;
     }
```

Comment:
Was this change done to reduce volume of logs?

Reply:
It was set at 800 in the constructor which I removed.  250 seemed a little aggressive, so I went with 1 second. You can change it in the constructor when you consume the package.